scripts/exrrfs_lbc.sh

```diff
@@ -46,11 +46,26 @@ EDATE=$($NDATE ${fhr_begin} ${CDATEin})
 start_time=$(date -d "${EDATE:0:8} ${EDATE:8:2}" +%Y-%m-%d_%H:%M:%S)
 EDATE=$($NDATE ${fhr_end} ${CDATEin})
 end_time=$(date -d "${EDATE:0:8} ${EDATE:8:2}" +%Y-%m-%d_%H:%M:%S)
-nvertlevels=55
-nsoillevels=4
+
+case ${MESH_NAME} in
+  conus12km)
+    nsoillevels=4
+    ;;
+  conus3km)
+    nsoillevels=9
+    ;;
+  *)
+    echo "unknow MESH_NAME=${MESH_NAME}"
+    export err=99; err_chk
+    ;;
+esac
```

Comment:
Soil levels are not related to the mesh. It is decided by the outside model.
If we use GFS to initialize the CONUS3km, the soil level should be 4 also.